apps/ledger-live-mobile/src/components/RecoverBanner/index.tsx

```diff
@@ -98,33 +98,45 @@ function RecoverBanner() {
         overflow="hidden"
         width="100%"
         onTouchEnd={onRedirectRecover}
-        p={3}
+        p="12px"
```

Comment:
Is the px value on purpose here ?

Reply:
I wanted to mention that also but @sshmaxime told me that there is a bug on that but i don't remember if it's on LLD or LLM

Reply:
Hi,
I have put value pixel instead of number because i have not good spacing on LLD.
Do you prefer i update with number ?